database/migrations/2022_06_17_181539_create_assignments_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('assignments', function (Blueprint $table) {
+            $table->id();
+            $table->bigInteger('course_id')->unsigned()->index()->nullable();
```

Comment:
Let's just change this to an unsignedBigInteger rather than having to tack on unsigned. You can also drop the index and nullable. Nullable we don't want because assignments should always be associated with a course. Indexing can definitely be helpful but let's not worry about it right now and we can study performance improvements at a later time.